sdv/sequential/par.py

```diff
@@ -76,13 +76,24 @@ def _get_context_metadata(self):
 
         for column in context_columns:
             context_columns_dict[column] = self.metadata.columns[column]
+            # Context datetime SDTypes for PAR have already been converted to float timestamp
+            if context_columns_dict[column]['sdtype'] == 'datetime':
+                context_columns_dict[column] = {'sdtype': 'numerical'}
```

Comment:
can we similarly pull this out into a generalized function?

Reply:
Done in [1127172](https://github.com/sdv-dev/SDV/pull/2347/commits/11271723d259276362bb24688d27b7df64d70c23)